src/apis/AppState/index.js

```diff
@@ -2,7 +2,7 @@ import ExecutionEnvironment from 'fbjs/lib/ExecutionEnvironment';
 import findIndex from 'array-find-index';
 import invariant from 'fbjs/lib/invariant';
 
-const EVENT_TYPES = [ 'change' ];
+const EVENT_TYPES = ['change'];
```

Comment:
A suggestion: if you have an editor that can work with JS styling profiles (Sublime or the like) you could tell it to load in code and automatically apply this one rule that makes your developer life easier, making sure that everything you load in has spaces, with a precommit hook that runs prettier on your src dir first, to ensure that the code you check in is what `prettier` generates. That way you get to enjoy the styling you like while you work on the code personally, but the project as a whole only ever sees prettier-"conformant" code.